packages/hoppscotch-sh-admin/src/helpers/userManagement.ts

```diff
@@ -0,0 +1,121 @@
+import { useToast } from '~/composables/toast';
+import { getI18n } from '~/modules/i18n';
+import { UserDeletionResult } from './backend/graphql';
+import { ADMIN_CANNOT_BE_DELETED, USER_IS_OWNER } from './errors';
+
+type IndividualActionInput = {
+  type: 'individual';
+  metadata: null;
+};
+type BulkActionInput = {
+  type: 'bulk';
+  metadata: {
+    areMultipleUsersSelected: boolean;
+    deletedIDs: string[];
+  };
+};
+
+type IndividualActionResult = {
+  data: null;
+};
+type BulkActionResult = {
+  data: { timeoutID: NodeJS.Timeout | null };
+};
```

Comment:
The `bulk` action type can return a `timeoutID` indicating the return type from `setTimeout` used to clear from the `/users` page. This is used to display the error messages after a certain delay following a series of toasts where the count for users deleted is displayed.